frontend/src/components/auth/Login.tsx

```diff
@@ -82,35 +103,58 @@ const Login = (): React.ReactElement => {
   }
 
   return (
-    <div style={{ textAlign: "center" }}>
-      <h1>Login</h1>
-      <form>
-        <div>
-          <input
-            type="email"
-            value={email}
-            onChange={(event) => setEmail(event.target.value)}
-            placeholder="username@domain.com"
-          />
-        </div>
-        <div>
-          <input
-            type="password"
-            value={password}
-            onChange={(event) => setPassword(event.target.value)}
-            placeholder="password"
-          />
-        </div>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onLogInClick}
-          >
-            Log In
-          </button>
-        </div>
-        <GoogleLogin
+    <Flex h="100vh">
+      <Box w="50%">
+        <Image src={KidsPicture} />
+      </Box>
+      <Flex w="50%" justifyContent="center">
+        <Flex
+          direction="column"
+          alignItems="center"
+          justifyContent="center"
+          maxWidth="450px"
+        >
+          <Image src={JumpMathLogo} alt="Jump Math Logo" w="296px" mb="24px" />
+          <Text textStyle="header4" mb="24px">
+            Admin Login
+          </Text>
+          <Text textStyle="subtitle2" mb={loginError ? "12px" : "48px"}>
+            Enter your login credentials below to continue
+          </Text>
+          {loginError && (
+            <Flex alignItems="center" pb="12px">
```

Comment:
TODO (once https://github.com/uwblueprint/jump-math/pull/88 is merged):  refactor to use `ErrorMessage.tsx` component